airflow/migrations/versions/0093_2_2_0_taskinstance_keyed_to_dagrun.py

```diff
@@ -226,7 +226,12 @@ def upgrade():
                 constraints = get_mssql_table_constraints(conn, "task_instance")
                 pk, _ = constraints["PRIMARY KEY"].popitem()
                 batch_op.drop_constraint(pk, type_="primary")
-            batch_op.drop_constraint("task_instance_pkey", type_="primary")
+            elif dialect_name == "sqlite":
```

Comment:
Mhm, you add the condition only for sqlite - which results in other DB engines are NOT dropping the constraint anymore - Or do I mis-interpret the code?

Reply:
If I'm not mistaken, I think https://github.com/apache/airflow/blob/b2769dcb9f51fc0e4c20ec22d4e9c3340475844b/airflow/migrations/versions/0093_2_2_0_taskinstance_keyed_to_dagrun.py#L226-L228 is already doing it for MySQL, and PostgreSQL is not handled elsewhere. So in the first commit, I removed the line `batch_op.drop_constraint("task_instance_pkey", type_="primary")` but later decide to let sqlite at least try.

Reply:
Or I can move `batch_op.drop_constraint("task_instance_pkey", type_="primary")` to the MySQL block if needed

Reply:
Mhm if I take a look to condition in line 225, this is ONLY mssql, not MySQL. Attempted to read it 3 times now... but as I read the code the change would lead to the result that in MySQL nothing is dropped. But the fix was for sqlite. So the change should not have a side-effect on other DB engines.

Reply:
Back in times of 2.2.0 we still had MSSQL as well :-D - acknowledged... just a few letters. We shoudl request they rename the DB engines in a way to better distinguish :-D

Reply:
Just updated it. First time using a mobile to update. Hope it works fine 🤔